platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less

```diff
@@ -376,6 +376,55 @@
 	}
 }
 
+.EmbedHtml {
+  margin-left: 26.5%;
+  margin-bottom: -5px;
+}
+
+.box {
+  background: #333333;
+  opacity: 90%;
+  margin-top: -5px;
+  display: flex;
+
+  .boxLink {
+    width: 50%;
+    padding: 7px;
+    margin-left: 8px;
+
+    .linkVideo {
+      width: 75%;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+
+      > a {
+        color: white;
+        font-family: 'Helvetica';
```

Comment:
Why did you put a font family here ? I think we should let the default one, from body tag